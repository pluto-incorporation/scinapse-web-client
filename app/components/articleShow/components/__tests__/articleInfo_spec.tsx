jest.unmock("../articleInfo");

import * as React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import ArticleInfo, { IArticleInfoProps } from "../articleInfo";

describe("<ArticleInfo /> component", () => {
  let articleInfoWrapper: ShallowWrapper<IArticleInfoProps>;
  const mockUser = {
    email: "sdfds",
    fullName: "MockUser",
    memberId: 2332,
  };
  const mockFrom = "Arxiv";
  const mockCreatedAt = "July 17, 2017";

  beforeEach(() => {
    articleInfoWrapper = shallow(<ArticleInfo from={mockFrom} createdBy={mockUser} createdAt={mockCreatedAt} />);
  });

  it("should match snapshot", () => {
    expect(articleInfoWrapper.html()).toMatchSnapshot();
  });
});
