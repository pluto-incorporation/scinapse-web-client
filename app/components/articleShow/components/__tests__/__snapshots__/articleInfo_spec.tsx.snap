// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArticleInfo /> component should match snapshot 1`] = `"<div class=\\"articleInfoWrapper\\"><span class=\\"articleInfoItem\\">from Arxiv</span><span class=\\"articleInfoItem\\">by tedyeee</span><span class=\\"articleInfoItem\\">posted at July 17, 2017</span></div>"`;
